README.md

```diff
@@ -165,3 +165,26 @@ function rex_url_shortener(rex_extension_point $ep) {
 	return $url;
 }
 ```
+
+
+## Weitere Tipps 
+
+### Leere Einträge vermeiden
+
+Werden URLs selbst erzeugt, z.B. über eine YForm-Tabelle, dann darf das Feld oder die Feldkombination, aus der die URL generiert wird, nur einmal vorkommen (prüfen auf `unique`) und außerdem niemals leer sein (prüfen auf `empty`).
```

Comment:
> dann darf das Feld oder die Feldkombination, aus der die URL generiert wird, nur einmal vorkommen (prüfen auf `unique`)

Das ist nicht richtig. Sofern die Url bereits vorhanden ist, wird beim nächsten die Id des Datensatzes angehangen.
Falls es das in der Version 2 nicht macht, wäre es ein Bug.

Reply:
In der Praxis will man dieses Verhalten vielleicht gar nicht erst provozieren, weil sich die URL des 2. Datensatz ändert, wenn man den 1. wegnimmt, oder nicht? Der Fallback ist gut, aber vielleicht nicht das, was man überhaupt haben will.

Reply:
Das ist richtig. Dann lassen wir es so. Danke.